packages/big-design/src/components/Modal/Modal.tsx

```diff
@@ -69,7 +69,7 @@ const InternalModal: React.FC<ModalProps> = ({
   const initialBodyOverflowYRef = useRef('');
   const internalTrap = useRef<FocusTrap | null>(null);
   const headerUniqueId = useUniqueId('modal_header');
-  const [modalRef, setModalRef] = useState<HTMLDivElement | null>(null);
+  const [modalRef, setModalRef] = useState<HTMLDivElement | HTMLElement | null>(null);
```

Comment:
Do we actually need this type?
```suggestion
  const [modalRef, setModalRef] = useState<HTMLDivElement | null>(null);
```

Reply:
`HTMLDivElement` extends from `HTMLElement` so it's pretty much the same thing.